### PR TITLE
Adding anchor js documentation

### PR DESCRIPTION
**Pull request by line47**
Adding to the documentation and cleaning up a bit of the documentation as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -239,16 +240,14 @@ After setting up your site on search.gov you can then add your `site handle` to
 You can add Google Analytics to your site by uncommenting the `google_analytics_ua` line and replacing `UA-????????-??` with your Google analytics UA code.
 
 ```
-# Configuration for Google Analytics, add your UA code here:
```
Comment (line47) [inline README.md]: Removing some of the extra commented sections here to just get at what need to be commented out

File: _config.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: This is the site title
 # This is used for adding an edit this page link to the footer
 github:
   organization: 18F
-  repository: federalist-uswds-template
+  repository: uswds-jekyll
```
Comment (line47) [inline _config.yml]: Updating the config to match the correct repo

File: README.md [hunk 3]
```diff
@@ -258,18 +257,26 @@ You can show the last date a page was last modified by uncommenting this line fr
 This will add the date right before the footer component and uses the `last-modified.html` include.
 
 ```yml
-# Used to show the "Last updated" date and time;
 # last_updated: true
 ```
 
+### Anchor JS
+
+You can show an anchor link next to header tags by uncommenting this section from the `_config.yml` data file.
```
Comment (jseppi) [inline README.md]: Maybe a note about what this is actually for would be good. 

"Anchor links will make individual section headers linkable" or something to that effect.

Comment (line47) [reply README.md]: Yes, great idea!

**Pull request merged**